cabal-install/src/Distribution/Client/Init/Interactive/Command.hs

```diff
@@ -306,7 +306,7 @@ cabalVersionPrompt flags = getCabalVersion flags $ do
     parseCabalVersion "2.4" = CabalSpecV2_4
     parseCabalVersion "3.0" = CabalSpecV3_0
     parseCabalVersion "3.4" = CabalSpecV3_4
-    parseCabalVersion _ = defaultCabalVersion -- 2.4
+    parseCabalVersion _ = defaultCabalVersion -- 3.4
```

Comment:
This comment looks like it's doomed to fall behind. Why don't we kill it, and anyone should be able to look up the definition of `defautCabalVersion` themselves?